The two E2E jobs fail here for a reason unrelated to this change: they cannot build the PWA image on `4.3` because of the broken `pnpm-lock.yaml` (`ERR_PNPM_BROKEN_LOCKFILE`, see #675). `Tests` and `Lint` are green. Rebasing on top of #675 once it lands will turn the E2E jobs green.

--- a/.github/workflows/check.yml
+++ b/.github/workflows/check.yml
@@ -20,10 +20,10 @@ jobs:
         name: Checkout
         uses: actions/checkout@v7
       -
+        name: Setup k6
+        uses: grafana/setup-k6-action@v1.2.1
+      -
         name: Run local k6 test
-        uses: grafana/k6-action@v0.3.1
-        with:
-          filename: k6/script.js
-          flags: --out json=results.json
+        run: k6 run k6/script.js --out json=results.json
         env:
           TARGET: ${{ inputs.url }}

